gdp/small_batch/gdp_small_batch.py

```diff
@@ -117,6 +282,17 @@ def coeffval_deact(disjunct):
 
     @m.Disjunction(m.k, m.j)
     def Y_exists_or_not(m, k, j):
+        """
+        Build the Logical Disjunctions of the GDP model for the small batch problem.
+
+        Args:
+            m (pyomo.ConcreteModel): small batch GDP model
+            k (int): number of parallel units
+            j (str): stage
+
+        Returns:
+            Disjunction
```

Comment:
This, in fact does not return a Disjunction. It returns a list with the two elements. Please correct and be more careful about this

Reply:
Corrected